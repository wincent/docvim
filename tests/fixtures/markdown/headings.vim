""
" @plugin foo We need this because otherwise we don't get link targets
"
" # A heading
"
" ## A subheading
"
" # An overlength heading when I really should have chose something shorter
"
" Note that in order for that last one to be shown as a heading in Vim, we can't
" allow the usual hard-wrapping rules to apply.
