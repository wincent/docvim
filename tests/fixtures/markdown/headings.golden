# foo<a name="foo-foo" href="#user-content-foo-foo"></a>

## A heading<a name="foo-a-heading" href="#user-content-foo-a-heading"></a>

### A subheading<a name="foo-a-subheading" href="#user-content-foo-a-subheading"></a>

## An overlength heading when I really should have chose something shorter<a name="foo-an-overlength-heading-when-i-really-should-have-chose-something-shorter" href="#user-content-foo-an-overlength-heading-when-i-really-should-have-chose-something-shorter"></a>

Note that in order for that last one to be shown as a heading in Vim, we can't allow the usual hard-wrapping rules to apply.
